MSAL/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>1C8F.1</string>
+				<string>CA92.1</string>
+			</array>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
+		</dict>
+	</array>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
```

Comment:
Should we mention also "NSPrivacyCollectedDataTypePhoneNumber" ?
More details here: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests#4250555

Reply:
Do you collect phone number in CIAM APIs?

Reply:
On native auth APIs we don't accept yet phone number as username. So, we don't collect it. But on web, if I'm not wrong, a phone number can be used as username. What happen in this case? Is the username (phone number) collected from the SDK? If not, you can resolve this conversation